src/components/__tests__/Bullet.ts

```diff
@@ -140,18 +141,22 @@ describe.skip('expansion', () => {
           - d
       `,
       }),
-      setCursor(['a', 'b']),
+      setCursor(unrankedPath),
     ])
 
-    const subthoughts = await findSubthoughts('a')
-    const bulletsOfSubthoughtsA = await findAllByLabelText(subthoughts[0], 'bullet')
-    userEvent.click(bulletsOfSubthoughtsA[0])
+    const context = contextToPath(store.getState(), unrankedPath)
```

Comment:
`contextToPath` converts a context to a Path, so it would be more appropriate to name this variable `path`. (The context is the input: `['a', 'b']`)